collections/misc/collprofiles.php

```diff
@@ -386,11 +402,11 @@ function directSubmitAction(e) {
 								if ($collManager->traitCodingActivated()) {
 									?>
 									<li>
-										<a href="#" onclick="$('li.traitItem').show(); return false;">
+										<a href="javascript:void(0)" onclick="showItemsList('traitItem')">
```

Comment:
It's not super common to see javascript:void(0) in the href attribute. Apparently, this works. But I had never seen it, and it sounds like just returning false at the end of the onclick is sufficient. E.g., `href="#" onclick="showItemsList('traitItem'); return false;">`

But I'll leave it up to you whether you want to make this change or not, since it seems to be working great!